src/validations/rules/ssp.sch

```diff
@@ -3467,6 +3483,16 @@
         </sch:rule>
     </sch:pattern>
     <sch:diagnostics>
+        <sch:diagnostic
+            doc:assertion="data-center-country-code"
+            doc:context="/o:location"
+            id="data-center-country-code-diagnostic">The data center address does not show a country.
```

Comment:
Suggest "The data center address does not specify a country code".

Reply:
Same comment as before.  But I should change the doc:assertion to be 'data-center-country'.